lib/compress.js

```diff
@@ -1739,22 +1739,22 @@ merge(Compressor.prototype, {
             "valueOf",
         ];
         var native_fns = {
-            Array: makePredicate([
+            Array: [
                 "indexOf",
                 "join",
                 "lastIndexOf",
                 "slice",
-            ].concat(object_fns)),
-            Boolean: makePredicate(object_fns),
-            Number: makePredicate([
+            ].concat(object_fns),
```

Comment:
Why are these concats necessary given that you have `convert_to_predicate(native_fns)`?